meson.build

```diff
@@ -5,6 +5,7 @@ project(
   license : 'GPL-3.0-or-later',
   default_options : ['cpp_std=c++17',],
 )
+add_project_arguments('-ffp-contract=off', language : 'cpp')
```

Comment:
I don't see this flag propagating to the meson build on aarch64
```
CFLAGS='-ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O3 -pipe -isy
stem /lsstsw/miniconda/envs/lsst-scipipe-8.0.0/include'
```

Reply:
Also not in any `CXXFLAGS` . Doesn't show up in the meson log.

Reply:
It should show up in `build-release/compile_commands.json`. Does it not?

Reply:
It does not.

Reply:
It does not show up on x86 either with an lsstsw build.

Reply:
Resolved in that I needed to add it to `python/meson.build` since the failures were from numpy image-based templated classes. It may not be necessary here but may as well stay for consistency, or if/when I end up adding GSL image classes.